contributing.md

```diff
@@ -92,54 +92,56 @@ As a general rule, we don't change public commit history, i.e. we don’t use ``
+In code review, the technical implementation of the bugfix or the new feature is reviewed. Changes during the review process are created in additional commits which are pushed to the feature branch. They are added to the existing pull request automatically. At the end of the code review, the reviewer approves the pull request and reassigns the pull request to its original creator.
 
 #### Done
 
-The creator of the pull request merges the pull request after checking the [definiton of done](#definition-of-done). After the merge, the issue and the linked pull request move to the done column automatically. We delete feature branches after merging.
+The creator of the pull request merges the pull request after checking the [Definiton of Done](#definition-of-done). After the merge, the issue and the linked pull request are closed and moved to the *Done* column automatically (due to the [closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) used in the pull request description). We delete feature branches after merging.
 
 ### Definition of Ready
 
-The *Definition of Ready* describes a list of criteria which tickets have to meet to move from column 'Backlog' to 'Ready':
+The *Definition of Ready* describes a list of criteria which issues have to meet to move from column 'Backlog' to 'Ready':
 
-- The person who implements the ticket is assigned and has every information to work on this ticket. Only the assignee can move the ticket to Ready.
-- It should be noted in the ticket by whom the ticket will be reviewed (e.g. add a comment like "will be reviewed by ...")
+- The person who will implement the issue is assigned and has every information to work on this issue. Only the assignee can move the issue to Ready.
+- The person who will review the issue is mentioned (e.g. add a comment like "could be reviewed by ...")
+- There are no blocking dependencies. Dependencies are expressed through simple referencing of the blocking issue (e.g. depends on #111), see details on [autolinked references and URLs](https://docs.github.com/en/github/writing-on-github/autolinked-references-and-urls)
 
 ### Definition of Done
 
-The *Definition of Done* describes a list of criteria which tickets have to meet to be called 'Done':
+The *Definition of Done* describes a list of criteria which issues have to meet to be called 'Done':
 
-- CI passed (CI contains tests)
-- Code reviewed
 - Functionality reviewed (approved by user/product owner)
-- Documentation (extern documentation must be linked)
\ No newline at end of file
+- Documentation exists (external documentation must be linked)
+- GitHub Actions / CI passed (contains tests)
+- Pull request is reviewed and approved
+- Functionality is merged into the main branch and deployed to production
```

Comment:
I think that 'deployed to production' should be removed since we have no production system yet.

Reply:
Right, pushed that in https://github.com/metafacture/metafacture-playground/pull/15/commits/6aac6404785dca902fc1d63aae2e0406f7b64aae